rfcs/0003-data-storage.md

```diff
@@ -0,0 +1,119 @@
+## Data Storage Structure
```

Comment:
Bikeshedding - Maybe start with heading 1? The same as rfc-02

Reply:
I mean change rfc-02 to start with heading 1, too >_<

Reply:
Make sense...